### PR TITLE
feat: added toast component

### PR DESCRIPTION
**Pull request by eakansh125**

### CODE REVIEW

**Review by gauravsinhaweb — APPROVED** [review]
Good to merge after changes.

File: toast.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                        <svg class="pointer" xmlns="http://www.w3.org/2000/svg" x="0px" y="0px" width="18" height="18" viewBox="0 0 50 50"
+                            style=" fill:#000000;">
+                            <path
+                                d="M 7.71875 6.28125 L 6.28125 7.71875 L 23.5625 25 L 6.28125 42.28125 L 7.71875 43.71875 L 25 26.4375 L 42.28125 43.71875 L 43.71875 42.28125 L 26.4375 25 L 43.71875 7.71875 L 42.28125 6.28125 L 25 23.5625 Z">
+                            </path>
+                        </svg>
+                    
+                    </div>
+                </div>
+
```
Comment (rajaranjanmr) [inline toast.html:84]: remove unnecessary spaces

File: style.css [hunk 2]
```diff
@@ -53,6 +53,10 @@ body{
     color: var(--primary-maroon);
 }
 
+.pointer{
+    cursor: pointer;
+}
+
```
Comment (rajaranjanmr) [inline style.css:95]: remove unnecessary spaces

**Review by rajaranjanmr — APPROVED** [review]
after changes in code, it can be merged. It seems god to me

File: toast.html (hunk 1 above)
Comment (eakansh125) [reply toast.html:84]: Sure

File: style.css (hunk 2 above)
Comment (eakansh125) [reply style.css:95]: Sure

Comment (eakansh125) [reply style.css:95]: Removed

**Pull request merged**